core/chunking.py

```
@@ -0,0 +1,467 @@
+    for root, _, files in os.walk(repo_path):
+        for fname in files:
+            ext = Path(fname).suffix.lower()
+            if ext in SUPPORTED_LANG:
+                file_path = Path(root) / fname
+                rel_path = file_path.relative_to(repo_path)
+                if spec and spec.match_file(str(rel_path)):
+                    continue
+                try:
+                    for chunk in chunker.chunk_file(file_path):
+                        yield chunk
+                except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)